tmt/steps/report/polarion.py

```diff
@@ -171,6 +171,16 @@ class ReportPolarionData(tmt.steps.report.ReportStepData):
              """
         )
 
+    fips: Optional[str] = field(
```

Comment:
```suggestion
    fips: bool = field(
```

unless you are planning on passing a specific value?

Reply:
Maybe even `is_flag=True`. As a boolean flag, it'd be a natural interface, `--fips` + `--no-fips` would be supported.

Plus a corresponding `default`, I suppose `default=False` would be acceptable.